docs/man/rpm-man-template.scd

```diff
@@ -24,9 +24,13 @@ Typographic conventions:
 - Use hard wrapping at 80 columns for all text
 - Use bold for literal values such as command name and options:
 *cmd* does X, with option *--do-this* does Y instead
-- Use bold when referring to section names
+- Use bold *UPPERCASE* for first level sections, and *Mixed case*
```

Comment:
I'd add "when referring to sections", otherwise it sounds like you should use bold for the sections themselves :sweat_smile:

Reply:
Yep that got a bit mixed up while editing: hopefully clarified now:

- Use UPPERCASE for first level sections, and Mixed case for second level sections (if present)
- Use matching case and bold when referring to section names

So the first bullet is for sections in general, the second is for references to them.

Reply:
Ohh, right, that makes much more sense, indeed :+1: